src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -1023,9 +1075,9 @@ def test_parse_log_and_stats_timeout(self):
         'corpus_size': 0,
         'crash_count': 0,
         'dict_used': 0,
-        'edge_coverage': 1321,
+        'edge_coverage': 0,
```

Comment:
they look valid changes as they crashed during seed corpus processing. i dont think we should have coverage metrics if fuzzing didn't start. wdyt @Dor1s @jonathanmetzman

Reply:
Makes sense

Reply:
Either way is fine, but this means we would see zero on the charts for e.g. super OOMy targets. Maybe we still want to see the coverage it is able to reach before crashing?